Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'pg', '~> 1.1'
 # Use Puma as the app server
 gem 'puma', '~> 5.0'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-# gem 'jbuilder', '~> 2.7'
+gem 'jbuilder', '~> 2.7'
```

Comment:
This was already added in the previous PR right?
Maybe you can do rebase so you get your last PR changes to this PR.. makes sense? Feel free to reach me if you have any doubts with this

Reply:
Yes, I did it, but it brought me the changes to this branch like new commits, so that's why it appears like "manually added"

Reply:
Oh, maybe you just need to update the base branch? Lets try that first

At the top, where it says `keinydev wants to merge 5 commits into CreateTarget from ShowTarget`
It should be the main branch instead of CreateTarget right?

Reply:
No, I create ShowTarget based on CreateTarget

Reply:
Yes you did. But `CreateTarget` was merged into `master`, right? So all changes we need here, are already in master.
So we can update the base branch of this PR.